### ISSUE TITLE
if only 1 fly, wing tracking fails

### ISSUE THREAD

**Issue by arobie** [kristinbranson/cbtrack_GUI#3]
If there are any bowls with only 1 fly, wing tracking fails and is omitted from results for all bowls.

**Comment by arobie** [comment]
I shared a exp dir with examples of 2 flies and 1 flies per bowl on dropbox. Both Shelby and I think this used to work ok in an earlier version. The expected behavior was for the program to count the flies and if there were 1 fly, track etc, if zeros flies just ignore. This is because its easiest for us to use the same ROIs for many movies and not have to delete ROIs with 1 or 0 flies. In the exp I shared, ROIs 3,6,8 and 9 have only 1 fly. I tried to set the fly count manually for 8 and 9. It looks like it tracked 2 flies in 3 and 6 and then failed on roi 8 and 9 in the choose orientation step.
